epsagon/trace.py

```diff
@@ -394,4 +485,4 @@ def send_traces(self):
 
 
 # pylint: disable=C0103
-tracer = Trace()
+factory = TraceFactory()
```

Comment:
Not very important, but I think that 'factory' is not a good name, especially when you import it directly so it's not clear that it is related to traces.

Reply:
Fixed